common/scripts/ci/run-sdk-backend-tiger-integration-tests-with-live-backend.sh

```diff
@@ -15,6 +15,7 @@ TIGER_DATASOURCES_NAME=$TIGER_DATASOURCES_NAME
 IS_AIO=$IS_AIO
 AIO_VERSION=$AIO_VERSION
 EXTRA_PARAMS=""
+IMAGE_ID=gooddata-cn-ce-aio-${EXECUTOR_NUMBER}
```

Comment:
should it be CONTAINER_ID?

Reply:
it's misleading, but the `CONTAINER_ID` and `IMAGE_ID` are the same

Reply:
so could you help to replace all IMAGE_ID by CONTAINER_ID for easier reading?

Reply:
Yes, working on that